images/build/php/prereqs/build.sh

```diff
@@ -83,6 +83,11 @@ else
 	apt-get install -y --no-install-recommends libargon2-dev
 fi
 
+if [ $PHP_MAJOR == '8' ]; then
+	apt-get update
+	apt-get install -y --no-install-recommends libonig-dev
```

Comment:
i see libonig-dev is already there for php 7.4+ .. so can you get rid of this extra if and use it at line 64?

Reply:
This line is necessary for 8.0 as well, I tried without it and it will fail because this library was not included.